shared/schemas/workflow-option-schema.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "WorkflowOption",
+    "description": "Workflow result option, key:value pairs.",
+    "type": "object",
+    "properties": {
+        "key": {
+            "description": "Unique identifier of the option. Preferably human-readable.",
+            "type": "string"
+        },
+        "value": {
+            "description": "Free form value of the option.",
+            "type": "string"
+        },
+        "valueType": {
+            "description": "More detailed type of the 'value' property. Defaults to 'text'.",
+            "enum": [
```

Comment:
I think it is wrong to have a free form type with just key-value pairs but try to enforce a type on each value.